src/components/FilterGroup.tsx

```diff
@@ -94,6 +100,11 @@ export function FilterGroup({
             />
           );
         })}
+        {limited &&
+        /* eslint-disable-next-line react-perf/jsx-no-new-function-as-prop */
+        <button className='text-primary py-1 text-sm' onClick={() => setShowAll(!showAll)}>
```

Comment:
Instead of using the eslint disable, can we extract the function and call it in onClick?

Reply:
I wanted to specifically use the eslint disable instead of using useCallback.

Extracting it to a function doesn't actually give it a stable reference, you have to use useCallback or pull the function out to the top level.
I'm fine with useCallback as well I just think it hurts performance a little because we don't actually need the memoization. But performance is not really a concern here!

Reply:
ohhh I see-- I totally missed that comment earlier. Thanks

Reply:
@yen-tt @nmanu1 do you guys prefer `useCallback` here? I'm really 50/50 and it's probably not a decision worth stressing over but just wondering

Reply:
I think I have a slight preference for not using `useCallback` here. just because it's not really needed and I'm in favor of not wrapping in `useMemo`/`useCallback` unless it adds some value